sdks/python/apache_beam/transforms/sql.py

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Transforms that use the SQL Expansion service."""
```

Comment:
The fact that it's using an SQL Expansion service is an implementation detail. Instead, I would document this as if they were natively written, e.g. "Transforms that allow one to perform SQL operations on Beam PCollections" with at least a cursory note about schemas, and perhaps an example (or at least pointer to the test as one), and then just a note that the actual implementation is shared with java.

Reply:
+1 The goal should be that these things work seamlessly and the docs should reflect that.